.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,31 @@
+name: Pull Request
+
+on:
+  push:
+    branches: ["main"]
+  pull_request: {}
+
+jobs:
+  changedfiles:
+    name: Publish to docs
+    runs-on: [self-hosted, services]
```

Comment:
```suggestion
    runs-on: [ubuntu-latest]
```

Reply:
ah, that would do it as well. thank you 😄